src/main/java/billgates/entities/UserFactory.java

```diff
@@ -0,0 +1,73 @@
+package billgates.entities;
+
+public class UserFactory {
+    //use getUser method to get object of type that implements Iuser
+    private int temporaryID;
+    private String temporaryName;
+    public String temporaryPW = "temporaryPW";
```

Comment:
Style: Please do not use abbreviations if possible.

Also, variable names here can be `password` instead of `temporaryPW`. I don't see a reason to add a `temporary` as a prefix.

Also, the variable here should be `private`. Then we use encapsulation to access this variable.

Reply:
Thank you for pointing this out! I will be more careful with style in future, thank you so much.
temporaryPW is because I want to distinguish the variable in two case (when we use the value stored in factory or the value we input when we call the getUser).